docs/vereisten/verplichtingen_van_aanbieders_van_ai_modellen_voor_algemene_doeleinden.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Verplichtingen van aanbieders van AI-modellen voor algemene doeleinden 
+toelichting: Aanbieders van AI-modellen voor algemene doeleinden moeten (technische) informatie en documentatie opstellen, up-to-date houden en beschikbaar stellen voor aanbieders van AI-systemen die het AI-model voor algemene doeleinden in hun AI-systemen willen integreren. 
+status_vereiste:
+- nog-niet-geldend
+levenscyclus:
+- ontwerp
+- dataverkenning-en-datapreparatie
+- ontwikkelen
+- verificatie-en-validatie
+- implementatie
+- monitoring-en-beheer
+- uitfaseren
+bouwblok:
+- transparantie
+hide:
+- navigation
+---
+
+<!-- tags -->
+## Vereiste
+
+Aanbieders van AI-modellen voor algemene doeleinden moeten (technische) informatie en documentatie opstellen, up-to-date houden en beschikbaar stellen voor aanbieders van AI-systemen die het AI-model voor algemene doeleinden in hun AI-systemen willen integreren.
+
+
+## Toelichting
+
+Aanbieders van AI-modellen voor algemene doeleinden hebben een bijzondere rol en verantwoordelijkheid binnen de AI-waardeketen.
+Zij leveren modellen die de basis kunnen vormen voor een reeks systemen verder in de AI-waardeketen, veelal aangeboden door aanbieders verder in de AI-waardeketen.
+Dit vraagt om een goed inzicht in de modellen en hun capaciteiten, zowel qua integratie van de modellen in hun producten als qua naleving van hun verplichtingen.
```

Comment:
Deze zinnen lezen een beetje verwarrend. Ook weet ik niet of bij iedereen de term AI-waardeketen bekend is. Dit is een vaag begrip en kunnen we denk ik concreter opschrijven. Misschien iets als:

```suggestion
Aanbieders van AI-modellen voor algemene doeleinden hebben een bijzondere rol en verantwoordelijkheid.
Zij leveren modellen die de basis kunnen vormen voor weer andere systemen en algoritmen, die vaak weer door andere partijen worden aangeboden dan de ontwikkelaar van het algemene systeem.
Dit vraagt om een goed inzicht in de modellen en hun capaciteiten, zowel qua integratie van de modellen in producten als qua naleving van verplichtingen.
```